connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -456,6 +457,24 @@ message ShowString {
   bool vertical = 4;
 }
 
+// Try eager evaluation.
+// It will invoke 'Dataset.showString' or 'Dataset.htmlString' to compose the result.
```

Comment:
I was wondering is it possible to implement the eager mode on client side only?

Reply:
Sure, we need another round-trip for retrieving some configs, though.
As for `_repr_html_`, we need another protocol anyway because it uses some methods that is currently not provided in Spark Connect.

Reply:
Yeah .. actually I think it needs a separate protobuf message at least for `_repr_html_`.